ReleaseNotes.md

```diff
@@ -1,5 +1,45 @@
 # Release Notes
 
+## New in Release 2.7.0
+
+Post-training Quantization:
+
+- Features:
+  - Added new types (AvgPool, GroupNorm, LayerNorm) to the ignored scope for `ModelType.Transformer` scheme.
+  - `QuantizationPreset.Mixed` was set as the default for `ModelType.Transformer` scheme.
+  - (OpenVINO) Added NF4-INT8, INT4 implementations for data-free WeightsCompression algorithm (`compress_weights(…)` pipeline).
+  - (OpenVINO) Added support for [IF operation](https://docs.openvino.ai/2023.1/openvino_docs_ops_infrastructure_If_8.html) quantization.
+  - (OpenVINO) Added [extract_ov_subgraph tool](tools/extract_ov_subgraph.py) for large IR subgraph extraction.
+  - (OpenVINO) Added `dump_intermediate_model` parameter support for AccuracyAwareAlgorithm (`quantize_with_accuracy_control(…)` pipeline).
+  - (OpenVINO) Added support for SmoothQuant and ChannelAlignment algorithms for HyperparameterTuner algorithm (`quantize_with_tune_hyperparams(…)` pipeline).
+  - (PyTorch) Post-training quantization implementation was replaced as the default.
```

Comment:
It could be like:
(PyTorch) Post-training Quantization is now supported with nncf.quantize() and the common implementation of quantization algorithms. Deprecated create_compressed_model() method for Post-training Quantization.

Reply:
Done